hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -668,6 +668,13 @@
       en_run_modes: ["sw_test_mode_test_rom"]
       reseed: 15
     }
+    {
+      name: chip_sw_lc_ctrl_raw_to_scrap
+      uvm_test_seq: chip_sw_lc_ctrl_raw_to_scrap_vseq
+      sw_images: ["//sw/device/tests/sim_dv:chip_sw_lc_ctrl_scrap_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      reseed: 1
```

Comment:
Hey @dkabely-ot could you add this line here:
```
      run_opts: ["+bypass_alert_ready_to_end_check=1"]
```
This will allow the test to exit with fatal alert.